.pre-commit-config.yaml

```diff
@@ -56,7 +56,7 @@ repos:
   - id: ruff
 
 - repo: https://github.com/psf/black
-  rev: "23.11.0"
+  rev: "23.10.0"
```

Comment:
possibly overlooked this, im not sure you meant to downgrade Black here?

Reply:
You're right.. I overlooked it, it shoudn't be downgraded